src/components/Footer/FooterMenuRow/FooterMenuRow.js

```diff
@@ -46,10 +46,10 @@ const FooterMenuRow = ({ data }) => (
         linkColumns={[
           [
             { title: 'Home', url: '/' },
-            { title: 'Mentors', url: '/' },
-            { title: 'Join us', url: '/' },
-            { title: 'Agenda', url: '/' },
-            { title: 'FAQ', url: '/' },
+            { title: 'About', url: '/about' },
+            { title: 'Join us', url: '/join' },
+            // { title: 'Agenda', url: '/agenda' },
```

Comment:
links already set in another MR, but mentors should still be removed

Reply:
Got it 👍